include/class.search.php

```diff
@@ -960,7 +960,7 @@ static function storeCounts($key, $counts, $ttl) {
     }
 
     static function clearCounts() {
-        if (function_exists('apcu_store')) {
+        if (extension_loaded('apc') && function_exists('apcu_store')) {
```

Comment:
I'm not so sure. On PHP >= 5.6 the extension is called `acpu`. So I think your fix is just disabling the block altogether? Because `function_exists('acpu_store')` should be enough to detect the presence of ACPu?

Reply:
`function_exists('apcu_store')` is not adequate enough to detect this as it fails for me every time when piping via CLI. I receive the error `APC must be enabled to use APCuIterator` which means APC is not enabled. So, I'm checking to make sure APC is enabled before continuing. I do see where this can be an issue so there has to be a better way other than detecting cli and this.